assets/js/util.js

```diff
@@ -45,6 +47,8 @@ function ajaxSend() {
 
 function ajaxComplete() {
     $('#loadingIndicator').hide();
+    apyRequestEndTime = Date.now();
+    checkServiceLoadTimes(apyRequestEndTime - apyRequestStartTime);
```

Comment:
Is there a need to store the end time? Why not just 

    checkServiceLoadTimes(Date.now() - apyRequestStartTime);